sparrow-py/pysrc/sparrow_py/_timestream.py

```diff
@@ -877,6 +877,22 @@ def shift_until(self, predicate: Timestream) -> Timestream:
         """
         return Timestream._call("shift_until", predicate, self)
 
+    def seconds_since_previous(self, n: int = 1) -> Timestream:
```

Comment:
We could have `e.seconds_since()` (from #652) automatically be seconds_since_previous.